Compiler/src/optimize.jl

```diff
@@ -1036,8 +1038,12 @@ matchpass(::Nothing, _, _) = false
 function run_passes_ipo_safe(
     ci::CodeInfo,
     sv::OptimizationState,
-    optimize_until = nothing,  # run all passes by default
+    optimize_until::Union{Nothing, Int, String} = nothing,  # run all passes by default
 )
+    if optimize_until isa String && !(optimize_until in ALL_PASS_NAMES)
```

Comment:
I'm not sure we have enough infrastructure in the compiler world to actually run this `in` test, but since we never call it from there anyway, I guess that's not a problem.

Reply:
I can change it to a loop?

Reply:
We can use `Base.Compiler.contains_is` for this. I will make the changes quickly.